en-us/resource-pack-issues.md

```diff
@@ -90,3 +78,18 @@ Beds:
 1. Close the game
 2. Remove or disable the Better Beds mod from your launcher.
 3. Start the game and see if it is fixed.
+
```

Comment:
IIRC broken paths and invisible blocks were connected issues, so maybe moving it further down doesn't make that much sense.

Reply:
https://github.com/RaptaG/wiki/commit/9c07b8cae6a178db2ca7195e729563bc1b050008